__tests__/app.test.js

```diff
@@ -115,21 +113,65 @@ describe("GET api/articles", () => {
         expect(body[0].votes).toBe(0);
         expect(body[0].article_img_url).toBe(
           "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
-          )
-          
-          expect(body).toBeSortedBy("created_at", { descending: true });
+        );
+
+        expect(body).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
+  test("Status 200: responds with an array of articles each containing a comments_count key with the total value of comments for said article", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body[0].comment_count).toBe("2");
+        expect(body[1].comment_count).toBe("1");
+        expect(body[12].comment_count).toBe("0");
+        expect(body[6].comment_count).toBe("11");
+      });
+  });
+});
+describe("GET api/articles/:article_id/comments", () => {
+  test("Status-200: responds with an array containing all comments for a given article ordered by most recent comment first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toBe(true);
+        expect(body[1]).toMatchObject({
+          comment_id: 2,
+          votes: 14,
+          created_at: "2020-10-31T03:03:00.000Z",
+          author: "butter_bridge",
+          body: "The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.",
+          article_id: 1,
         });
+
+        expect(body).toBeSortedBy("created_at", { descending: true });
```

Comment:
we can do this in its own test to split up the behaviours - one test for checking the properties and one to check the sorted order